scripts/deno/thrasher-tracker.ts

```diff
@@ -1,10 +1,12 @@
 import { array, object, string } from 'https://deno.land/x/zod@v3.17.3/mod.ts';
 import { fetchJSON } from './json.ts';
-import prettyBytes from 'https://esm.sh/pretty-bytes';
+import prettyBytes from 'https://esm.sh/pretty-bytes@6.0.0';
 import { octokit } from './github.ts';
 
 // -- Constants -- //
 
+const gu = new URL('https://www.theguardian.com/');
```

Comment:
Does this need to be a URL object? I didn't know you could pass in `new URL()` to itself but it seems you can. But do we need to if we're hard coding the string like this?

Reply:
![Your scientists were so preoccupied with whether or not they could, they didn’t stop to think if they should.](https://user-images.githubusercontent.com/76776/187410615-b68f6161-c1f0-456e-a2e3-7693f792ca8d.png)